test/data/codegraph/simple.c

```
@@ -0,0 +1,12 @@
+#include<stdio.h>
+
+int a(int x)
```

Clang (14.0.6)

[warning] parameter 'x' is unused

(misc-unused-parameters)